From: Sebastian Noack <sebastian.noack@gmail.com>
Date: Thu, 18 May 2017 10:13:40 +0200
Subject: Default to x-terminal-emulator instead hard-coding urxvt

---
 config.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config.in b/config.in
index b39a39a..b75d65d 100644
--- a/config.in
+++ b/config.in
@@ -14,7 +14,7 @@ set $down j
 set $up k
 set $right l
 # Your preferred terminal emulator
-set $term urxvt
+set $term x-terminal-emulator
 # Your preferred application launcher
 set $menu dmenu_run
 
